packages/storage-documents/src/page-scan-run-result.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { StorageDocument } from '.';
+
+export declare type ReportFormat = 'sarif';
+export declare type ScanState = 'unknown' | 'pass' | 'fail';
+export declare type RunState = 'unknown' | 'accepted' | 'queued' | 'running' | 'completed' | 'failed';
+
+/**
+ * The web page scan run result document.
+ */
+export interface PageScanRunResult extends StorageDocument {
```

Comment:
we have same types defined in 2 packages?

Reply:
The web-api interface is for client response that is decoupled from a storage version that is different and may change w/o affecting client REST API interface